pkg/kubelet/network/dns/dns.go

```diff
@@ -384,7 +364,7 @@ func appendDNSConfig(existingDNSConfig *runtimeapi.DNSConfig, dnsConfig *v1.PodD
 
 // GetPodDNS returns DNS settings for the pod.
 func (c *Configurer) GetPodDNS(pod *v1.Pod) (*runtimeapi.DNSConfig, error) {
-	dnsConfig, err := c.getHostDNSConfig()
+	dnsConfig, err := c.getHostDNSConfig(c.ResolverConfig)
```

Comment:
nit for followup: maybe we should rename this field to `ResolverConfigPath` or something - I had to go look it up :)

Reply:
Done: https://github.com/kubernetes/kubernetes/pull/116547

Reply:
Will abandon that PR.